### DISCUSSION TITLE
Overall Design Elements

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#173] [Ideas]
I assume we are looking at this for a better site than (SQL Saturday)[https://www.sqlsaturday.com/]. With that in mind, perhaps we need a list of the items to deal with? This list likely needs to be on the wiki, but a discussion here first would be good.

- main site for listing events
  - list of upcoming events
  - search for events by various means (location, name, status, etc.)
  - feed of upcoming events
  - global sponsors (logo, text, link)
  - submission for a new event
  - general about/faq/support/etc.
- individual sites/pages for each event
  - data managed by an organiser (description, date, location, etc.
  - Code of Conduct - ??? do we allow customisations/additions ???
  - T&C - ??? customisations/additions???
  - email features to contact speakers/attendees/volunteers
  - schedule of sessions with locations
  - support for multiple days (pre-cons/post-conts, day of) -- ??? Multiple day events???
  - attendee registration
  - cart for adding pre/post con and lunch
  - integration with payment options
  - feed for mobile apps   -- ??? do we add a mobile app to the project (??separate project??)
  - sponsor list
  - networking/who is attending
  - support for speaker uploads (?? zip only??, ??separate PPT/code??)
  - ical
  - support for online platforms (separate urls)
  - map app support - one thing I find hard from SQLSat is easily getting directions day of
  - news/updates from organisers
  - volunteer requests/opportunities
  - evals
  - list of speakers, links to some bio/speaking cv/site/etc.
  - build a schedule?
- Admin side
  - approve/decline sessions
  - manage waitlist
  - manage refunds (easy link to payment sites)
  - produce badges/speedpass/tickets
  - manage sponsor invoices/AR features
  - manage indididual registrations
  - graphing for data elements
  - social media promotion (scheduling)
  - email promotion (scheduling)
  - todo list
  - burndown of milestones to events, ical integration?
  - reminders for groups (speakers, organizers, etc.)
  - templates for emails and social media promotions (token by event?)

**Comment by sqlzelda** [comment]
One thing that's been mentioned in other issues, is the ability to reach out to attendees that checked in at the site. Do we want to consider allowing this kind of functionality especially with a bar code like Rob Volk and others were working to use. 

I've also wondered if we need to start embracing virtual/digital aspects of events. We've proven over the past several months that these types of events can happen. Allowing for remote attendance has also been highlighted as an accessibility issue. I understand that may bring up questions like closed captioning as well.

I'm not saying we have to have the answers today, but I would like it if we could decide if remote options are something we'd like to plan for in the future.
